Packs/AWS/Integrations/AWS/AWS.py

```diff
@@ -2032,6 +2118,216 @@ def describe_ipam_resource_discovery_associations_command(client: BotoClient, ar
         )
         return command_results
 
+    @staticmethod
+    def get_latest_ami_command(client: BotoClient, args: Dict[str, Any]) -> CommandResults:
+        """
+        Retrieves information about the latest Amazon Machine Image (AMI) based on provided filters.
+        The function calls the AWS EC2 'describe_images' API, sorts the results by
+        'CreationDate' in descending order, and returns the first image in the list.
+        Args:
+            client (BotoClient): The initialized Boto3 EC2 client.
+            args (Dict[str, Any]): Command arguments, typically containing:
+                  Expected keys include 'executable_by', 'filters', 'owners', 'image_id',
+                  'include_deprecated', 'include_disabled', 'limit', and 'next_token'.
+
+        Returns:
+            CommandResults: An object containing the raw image details as outputs,
+                            and a md table with a concise summary of the latest AMI.
+        """
+        kwargs = {
+            "ExecutableUsers": parse_resource_ids(args.get("executable_users")) if args.get("executable_users") else None,
+            "Filters": parse_filter_field(args.get("filters")),
+            "Owners": parse_resource_ids(args.get("owners")) if args.get("owners") else None,
+            "ImageIds": parse_resource_ids(args.get("image_ids")) if args.get("image_ids") else None,
+            "IncludeDeprecated": arg_to_bool_or_none(args.get("include_deprecated")),
+            "IncludeDisabled": arg_to_bool_or_none(args.get("include_disabled")),
+            "MaxResults": args.get("limit"),
+            "NextToken": args.get("next_token"),
+        }
+
+        remove_nulls_from_dictionary(kwargs)
+        pagination_kwargs = build_pagination_kwargs(kwargs)
+        response = client.describe_images(**pagination_kwargs)
```

Comment:
```suggestion
        kwargs = {
            "ExecutableUsers": parse_resource_ids(args.get("executable_users")) if args.get("executable_users") else None,
            "Filters": parse_filter_field(args.get("filters")),
            "Owners": parse_resource_ids(args.get("owners")) if args.get("owners") else None,
            "ImageIds": parse_resource_ids(args.get("image_ids")) if args.get("image_ids") else None,
            "IncludeDeprecated": arg_to_bool_or_none(args.get("include_deprecated")),
            "IncludeDisabled": arg_to_bool_or_none(args.get("include_disabled")),
        }

        remove_nulls_from_dictionary(kwargs)
        pagination_kwargs = build_pagination_kwargs(kwargs)
        pagination_kwargs.update(kwargs)
        response = client.describe_images(**pagination_kwargs)
```
When using build_pagination_kwargs(), there is no need to add MaxResults and NextToken to kwargs.
The return value of build_pagination_kwargs() includes only the pagination arguments without the kwargs, you need to merge them to get the full dict of arguments.